polkadot-parachains/statemine-runtime/src/lib.rs

```diff
@@ -638,6 +638,7 @@ impl pallet_aura::Config for Runtime {
 parameter_types! {
 	pub const PotId: PalletId = PalletId(*b"PotStake");
 	pub const MaxCandidates: u32 = 1000;
+	pub const MinCandidates: usize = 1;
```

Comment:
Maybe 5?

Reply:
ya, I had no idea, that sounds fine to me, although one small thing is that it won't allow people to exit with their deposits until others join in (if they are below the 5 limit), which seems fine to me I just want to bring it up

Reply:
actually gonna add it to the pallet documentation with the amount change